README.md

```diff
@@ -34,6 +36,32 @@ As the enhancement proposal reveals, this is not a fully compliant CSI Driver im
 solely provides the minimum amounts of the Kubernetes / CSI contract needed to achieve the goals stipulated in the 
 Enhancement proposal.
 
+## Current status with respect to the Kubernetes CSIVolumeSource API
+
+So let's take each part of the [CSIVolumeSource struct](https://github.com/kubernetes/api/blob/71efbb18d63cd30604981514ac623a6be1d413bb/core/v1/types.go#L1743-L1771):
```

Comment:
It's opinionated, but I would remove `struct` word from the url, because it is not part of the entity, but rather narrowing description of the entity. It can be removed completely, and text meaning will not be changed. While not being part of the entity name, I would remove it from the url leading to the entity.

Reply:
I agree ... part of next push

Reply:
I'm also going to clarify outside of the url with something like "Kubernetes API"

Reply:
check that I already do that in the title